app/models/option.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: options
+#
+#  id                      :integer          not null, primary key
+#  created_at              :datetime         not null
+#  updated_at              :datetime         not null
+#  demographic_question_id :integer
+#
+# Indexes
+#
+#  index_options_on_demographic_question_id  (demographic_question_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (demographic_question_id => demographic_questions.id)
+#
+
+class Option < ApplicationRecord
+  belongs_to :demographic_question, optional: true
```

Comment:
Since the relationship is the other way around we can remove the `optional: true` here, because an option will always belong to a demographic question

Reply:
Ah sure thanks @spencerldixon